src/components/FilterSearch.tsx

```diff
@@ -93,9 +96,11 @@ export function FilterSearch({
     if (filter && displayName) {
       answersActions.setFilterOption({ ...filter, displayName, selected: true });
       answersActions.setOffset(0);
-      executeSearch(answersActions);
+      if (searchOnSelect) {
+        executeSearch(answersActions);
```

Comment:
for `searchOnSelect=false`, would people use the `ApplyFiltersButton` to execute the search like they do for facets and static filters?